src/utils/functions/getNotarizationSignatures.ts

```diff
@@ -0,0 +1,58 @@
+import { NotaryAsset } from "../../types/notary-details"
+import fetchNotarizationFromEndPoint from "./fetchNotarizationFromEndPoint"
+
+/**
+ * Get notarization signatures from a notary endpoint defined in the asset object, see spec for more info: https://github.com/syscoin/sips/blob/master/sip-0002.mediawiki
+ * @param notaryAssets Required. Asset objects that require notarization, fetch signatures via fetchNotarizationFromEndPoint()
+ * @param txHex Required. Signed transaction hex created from syscointx.createTransaction()/syscointx.createAssetTransaction()
+ * @returns boolean representing if notarization was done by acquiring a witness signature from notary.
+ */
+export async function getNotarizationSignatures (notaryAssets: Map<string, NotaryAsset>, txHex: string): Promise<boolean> {
```

Comment:
Lets temporarily add ESLint disables as we don't want to alter the logic for this yet.

```typescript
/* eslint-disable no-await-in-loop */
/* eslint-disable no-continue */
/* eslint-disable no-restricted-syntax */
```